content/applications/hr/employees/new_employee.rst

```diff
@@ -183,42 +195,48 @@ this section may be critical for the company's payroll department. In order to p
+  employee's license plate information in the :guilabel:`Private Car Plate` field.
 - :guilabel:`Education`: Select the highest level of education completed by the employee from the
   :guilabel:`Certificate Level` drop-down menu. Options include :guilabel:`Graduate`,
   :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, or :guilabel:`Other`. Type in the
   :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective fields.
+- :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
+  Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.
+  Using the calendar module, select the :guilabel:`Visa Expire Date` and/or the :guilabel:`Work
+  Permit Expiration Date` to enter the expiration date(s). If available, upload a digital copy of
+  the work permit document. Click :guilabel:`Upload Your File`, navigate to the work permit file in
+  the file explorer, and click :guilabel:`Open`.
+- :guilabel:`Family Status`: Select either :guilabel:`Single`, :guilabel:`Married`,
+  :guilabel:`Legal Cohabitant`, :guilabel:`Widower`, or :guilabel:`Divorced` from the drop-down
+  menu. If the employee has any dependent children, enter the :guilabel:`Number of Dependent
+  Children` in the field.
+- :guilabel:`Emergency`: Type in the name and phone number of the employee's emergency
+  contact.
 - :guilabel:`Citizenship`: This section houses all the information relevant to the citizenship of
   the employee. Some selections use a drop-down menu, as does the :guilabel:`Nationality (Country)`,
   :guilabel:`Gender`, and :guilabel:`Country of Birth` sections. The :guilabel:`Date of Birth` uses
   a calendar module to select the date. First, click on the name of the month, then the year, to
   access the year ranges. Use the :guilabel:`< (left)` and :guilabel:`> (right)` arrow icons,
   navigate to the correct year range, and click on the year. Next, click on the month. Last, click
   on the day to select the date. Type in the information for the :guilabel:`Identification No`,
-  :guilabel:`Passport No`, and :guilabel:`Place of Birth` fields. If the employee is
-  :guilabel:`Disabled` or a :guilabel:`Nonresident`, click the check box next to the respective
-  fields.
-- :guilabel:`Dependant`: If the employee has any dependants, that information is entered here. Type
-  in the number of children the employee has, and check the boxes next to :guilabel:`Disabled
-  Children` and/or :guilabel:`Other Dependent People` if applicable.
-- :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
-  Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.
-  Using the calendar module, select the :guilabel:`Visa Expire Date` to enter the expiration date.
+  :guilabel:`Passport No`, and :guilabel:`Place of Birth` fields.
```

Comment:
The "No"s in this section also need periods at the end, if you're submitting feedback.

Reply:
I will suggest they add this- I am leaving this as you see it, since this is how it currently appears on the UI.